providers/src/airflow/providers/microsoft/azure/hooks/adx.py

```diff
@@ -41,7 +41,7 @@
 )
 
 if TYPE_CHECKING:
-    from azure.kusto.data.response import KustoResponseDataSetV2
+    from azure.kusto.data.response import KustoResponseDataSet
```

Comment:
What is the difference between `KustoResponseDataSetV2` and `KustoResponseDataSet`?

Reply:
As per the azure-kusto-python repo: https://github.com/Azure/azure-kusto-python/blob/9e86492124db7361d0ba21943f8839d1dc12d6d3/azure-kusto-data/azure/kusto/data/response.py#L139-L151, KustoResponseDataSetV2 is a wrapper on KustoResponseDataSet for some response wise filtering

Reply:
Resolving the comment to merge and unblock CI, please reopen in case you have an issue.